charts/lightstepsatellite/README.md

```diff
@@ -29,7 +29,7 @@ Lightstep microsatellite to collect telemetry data.
 | lightstep.guid | string | `nil` | defaults to pod's name using the Downward API |
 | lightstep.http_plain_port | int | `8181` |  |
 | lightstep.http_secure_port | int | `9191` |  |
-| lightstep.max_msg_size_bytes | int | `0` | Max gRPC server message size the microsatellite will accept. A value of 0 means the gRPC default of 4MiB |
```

Comment:
can you please put this back?

Reply:
the PR checks failed which is why I changed it... let me take a closer look

Reply:
Should be fixed now with Greg's change: https://github.com/lightstep/lightstep-microsatellite-helm-chart/pull/18